### ISSUE TITLE
Unable to get test details in extentX server

### ISSUE THREAD

**Issue by brianjmitchell** [anshooarora/extentx#95]
I am trying to get this working as this would be pefect for collecting the many test results i have planned.

At the moment i have mongoDB running along with extentX, i am getting some data to the extentX server but none of the tests information is here.

![image](https://cloud.githubusercontent.com/assets/8811602/21260941/6619bb98-c380-11e6-8ff3-c3f293613f51.png)

You can see the server is being updated with tests, but going into each test i see nothing.

![image](https://cloud.githubusercontent.com/assets/8811602/21260957/81ce9d5e-c380-11e6-978e-28a07a835820.png)

Here is the extent report that is being created - extentReports.html

![image](https://cloud.githubusercontent.com/assets/8811602/21260980/a623a104-c380-11e6-80fb-21a716f8c55b.png)
![image](https://cloud.githubusercontent.com/assets/8811602/21261188/ca498c28-c381-11e6-920e-1c35922e22cb.png)

The issue I am getting is this info is not being sent up to extentX server, am I doing something wrong or missing a step.

Here is my a my testNG test base to help you understand a little.

public class TestBase extends TestListenerAdapter {

    private String uri;
    protected Logger APPLICATION_LOGS = LoggerFactory.getLogger(getClass());
    public ThreadLocal<ExtentTest> parentTest = new ThreadLocal<ExtentTest>();
    public ThreadLocal<ExtentTest> test = new ThreadLocal<ExtentTest>();
    public static ExtentTest childTest;
    public ExtentTest parent;
    public ExtentTest child;

    protected String getUri() {
        return uri;
    }

    @BeforeSuite
    @Parameters({"env", "testName"})
    public void initialiseReport(String env, String testName) {

        ExtentManager.prop.setProperty("projectName", testName);
    }

    @BeforeClass
    @Parameters({"protocol", "env", "server", "port", "endPoint", "testName"})
    public void initialiseURI(String protocol, String env, String server, String port, String endPoint, String testName) {
        uri = protocol + env + "." + server + ".mcom:" + port + "/" + endPoint;
        APPLICATION_LOGS.debug("Will use the following URI: " + uri);

        baseURI = uri;
        given().when().post().then().statusCode(200);

        createParentNodeExtent(testName, "This is a description");
    }

    @BeforeMethod
    public final void beforeMethod(Method method) {
        createChildNodeExtent(method);
    }

    @AfterMethod
    public void afterMethod(ITestResult result) {
        switch (result.getStatus()) {
            case ITestResult.FAILURE:
                test.get().fail(result.getThrowable());
                test.get().log(Status.FAIL, "<pre>" + result.getThrowable() + "</pre>");
                break;
            case ITestResult.SKIP:
                test.get().skip(result.getThrowable());
                break;
            case ITestResult.SUCCESS:
                test.get().pass("Test passed");
                test.get().log(Status.PASS, result.getMethod().getMethodName());
                break;
            default:
                break;
        }
    }

    @AfterClass
    public void afterClass(){
        ExtentManager.getExtent().flush();
    }

    @AfterSuite
    public void completeReport() {
    }

    public void createParentNodeExtent(String methodName, String testDescription) {
        parent = ExtentTestManager.createTest(methodName, testDescription);
        parentTest.set(parent);
        ExtentTestManager.getTest().log(Status.INFO, "this is a parent");
    }

    public void createChildNodeExtent(Method method){
        child = parentTest.get().createNode(method.getName());
        childTest = child;
        test.set(child);
    }
}

The only logging in the tests I am performing is using syntax like

childTest.log(Status.INFO, "<b>Syndicator Details: ID: </b>" + syndicatorID);

To sum up I am hosting an extentX server and the extent report I am creating is not being shown on the server.

Any help is really appreciated as I have spent a lot of time trying to get this working.

**Comment by brianjmitchell** [comment]
I was using the wrong extentX distribution...........................this took me 1.5 days to figure out!!
